app/src/hooks/api/useStandardsApi.ts

```diff
@@ -12,16 +12,54 @@ const useStandardsApi = (axios: AxiosInstance) => {
   /**
    * Fetch species standards
    *
-   * @return {*}  {Promise<IGetSpeciesStandardsResponse>}
+   * @return {*}  {Promise<ISpeciesStandards>}
    */
-  const getSpeciesStandards = async (tsn: number): Promise<IGetSpeciesStandardsResponse> => {
+  const getSpeciesStandards = async (tsn: number): Promise<ISpeciesStandards> => {
     const { data } = await axios.get(`/api/standards/taxon/${tsn}`);
 
     return data;
   };
 
+  /**
+   * Fetch method standards
+   *
+   * @param {string} keyword
+   * @return {*}  {Promise<IGetMethodsStandardsResponse>}
+   */
+  const getMethodStandards = async (keyword?: string): Promise<IMethodStandard[]> => {
+    let url = '/api/standards/methods';
+
+    if (keyword) {
```

Comment:
You might be able to use `paramsSerializer`
Check useObservationApi.ts for reference.